web/angular-wallet/src/app/main/dashboard/balance/balance.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { BalanceComponent } from './balance.component';
+
+describe.skip('BalanceComponent', () => {
```

Comment:
no time yet... I'm in a hurry...need to do this later (I know, the biggest lie in SW dev)